src/main/kotlin/io/thecontext/ci/output/HtmlRenderer.kt

```diff
@@ -0,0 +1,16 @@
+package io.thecontext.ci.output
+
+import org.commonmark.parser.Parser
+
+interface HtmlRenderer {
+
+    fun render(markdown: String): String
```

Comment:
NIT: why gets a HtmlRendere markdown as a String? Isn't it an implementation detail and should be of type episode (and transform it under the hood to markdown then to html)

Reply:
This way `HtmlRenderer` doesn’t know about outside world types. Single responsibility, basically.